module/x/gravity/types/key.go

```diff
@@ -260,7 +260,7 @@ func GetAttestationKey(eventNonce uint64, claimHash []byte) string {
 // [0x6][0xc783df8a850f42e7F7e57013759C285caa701eB6]
 // This prefix is used for iterating over unbatched transactions for a given contract
 func GetOutgoingTxPoolContractPrefix(contractAddress EthAddress) string {
-	return OutgoingTXPoolKey + contractAddress.GetAddress()
+	return ConvertByteArrToString(append([]byte(OutgoingTXPoolKey), contractAddress.GetAddress().Bytes()...))
```

Comment:
Do we want to use this function here? Are we unable to use string(...)?

Reply:
Ah, I didn't realize I've added this 🤦 I'll remove it right away